app/src/main/java/com/framgia/forder/screen/createshop/CreateshopFragment.java

```diff
@@ -3,10 +3,14 @@
 import android.app.Activity;
 import android.content.Intent;
 import android.databinding.DataBindingUtil;
+import android.graphics.Bitmap;
```

Comment:
ERROR: Unused import: android.graphics.Bitmap. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports